### PR TITLE
Np 1979 migrate CreatePublication and ModifyPublication handlers

### PR DESCRIPTION
**Pull request by axthosarouris**
All merges will happen to branch "refactoring" so that development will continue working.

### CODE REVIEW

File: modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java [hunk 1]
```diff
@@ -195,42 +181,37 @@ private TestAppender createAppenderForLogMonitoring() {
...
 
     private void serviceSucceedsAndReturnsModifiedPublication(Publication modifiedPublication)
         throws ApiGatewayException {
         serviceSucceedsOnGetRequest(publication);
-        when(publicationService.updatePublication(any(SortableIdentifier.class), any(Publication.class)))
+        when(publicationService.updatePublication(any(Publication.class)))
             .thenReturn(modifiedPublication);
     }
 
-    private void serviceSucceedsOnGetRequestAndFailsOnUpdate() throws ApiGatewayException {
```
Comment (axthosarouris) [inline modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java:228]: The ErrorResponeException is not thrown by any method of the current DynamoDB service

File: modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java [hunk 2]
```diff
@@ -136,18 +134,6 @@ public void handlerReturnsBadRequestResponseOnMissingPathParam() throws IOExcept
         assertThat(getProblemDetail(gatewayResponse), containsString(IDENTIFIER_IS_NOT_A_VALID_UUID));
     }
 
-    @Test
```
Comment (axthosarouris) [inline modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java:139]: This test refers to an exception that is never thrown by the service

File: template.yaml [hunk 3]
```diff
@@ -125,7 +125,7 @@ Resources:
                     $ref: '#/definitions/Publication'
               x-amazon-apigateway-integration:
                 uri:
-                  Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCreatePublicationFunction.Arn}/invocations
+                  Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCreatePublicationFunction.Arn}:live/invocations
```
Comment (axthosarouris) [inline template.yaml:128]: Added an example of a hot instance.

File: template.yaml [hunk 4]
```diff
@@ -389,29 +366,29 @@ Resources:
...
+      MinCapacity: 1
+      ResourceId: !Sub function:${NvaCreatePublicationFunction}:live # You need to specify an alis or version here
+      RoleARN: !Sub arn:aws:iam::${AWS::AccountId}:role/aws-service-role/lambda.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_LambdaConcurrency
+      ScalableDimension: lambda:function:ProvisionedConcurrency
+      ServiceNamespace: lambda
+    DependsOn: NvaCreatePublicationFunctionAliaslive # This is your function logical ID + "Alias" + what you use for AutoPublishAlias
 
-  NvaCreateResourceFunction:
-    Type: AWS::Serverless::Function
+  MyTargetTrackingScalingPolicy:
```
Comment (axthosarouris) [inline template.yaml:380]: The name changes in next PR

File: template.yaml [hunk 5]
```diff
@@ -380,7 +357,7 @@ Resources:
           API_HOST: !Ref CustomDomain
       Policies:
         - DynamoDBCrudPolicy:
-            TableName: !Ref PublicationsTableName
+            TableName: !Ref OrestisResources
```
Comment (brinxmat) [inline template.yaml:360]: ?

File: template.yaml [hunk 6]
```diff
@@ -449,12 +426,12 @@ Resources:
       Environment:
         Variables:
           ALLOWED_ORIGIN: '*'
-          TABLE_NAME: !Ref PublicationsTableName
+          TABLE_NAME: !Ref OrestisResources
```
Comment (brinxmat) [inline template.yaml:429]: I guess these change in the next PR too

**Review by brinxmat — APPROVED** [review]

**Pull request merged**